localization/util/pose_initializer/README.md

```diff
@@ -0,0 +1,24 @@
+# pose_initializer
+
+## Role
```

Comment:
```suggestion
## Purpose
```
IMO: The doc template and the other READMEs use `Purpose`. <https://github.com/tier4/AutowareArchitectureProposal.iv/blob/ff7368ec6a2ee596071f10cfdb3bfa5a7e48dba0/design_document_template.md?plain=1#L3>
If it is intentional, please ignore this comment.

Reply:
Thank you. will rename it.

Reply:
6dd5632 Thanks! Resolved.